code/controllers/subsystem/traitor.dm

```diff
@@ -51,6 +53,30 @@ SUBSYSTEM_DEF(traitor)
 				log_world("[configuration_path] has an invalid type ([typepath]) that doesn't exist in the codebase! Please correct or remove [typepath]")
 			configuration_data[actual_typepath] = data[typepath]
 
+	/// List of high-security areas that we pick required ones from
+	var/list/allowed_areas = typecacheof(list(/area/station/command,
+											  /area/station/cargo/qm,
+											  /area/station/comms,
+											  /area/station/engineering,
+											  /area/station/science,
+											  /area/station/security,
+											  ))
```

Comment:
No alignment indentation

https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#tabs-not-spaces

Reply:
Then make sure `))` is on one level lower